templates/web/base/auth/general.html

```diff
@@ -77,61 +74,27 @@ <h3>[% tprintf(loc("Do you have a %s password?", "%s is the site name"), site_na
 [% INCLUDE 'footer.html' %]
 
 [% BLOCK form_sign_in_yes %]
-    <div id="form_sign_in_yes" class="form-box">
-        <h5>[% loc('<strong>Yes</strong> I have a password') %]</h5>
-
+    <div class="hidden-nojs js-sign-in-password-hide">
+        <input class="btn btn--sign-in-password js-sign-in-password-btn" type="submit" name="sign_in_by_password" value="[% loc('Sign in with a password') %]">
+    </div>
+    <div class="hidden-js js-sign-in-password">
         <label for="password_sign_in">[% loc('Your password') %]</label>
 
         <div class="form-txt-submit-box">
-            <input type="password" name="password_sign_in" class="form-control" id="password_sign_in" value="">
+            <input type="password" name="password_sign_in" class="form-control" id="password_sign_in" value="" autocomplete="current-password">
             <input class="green-btn" type="submit" name="sign_in_by_password" value="[% loc('Sign in') %]">
         </div>
 
-        <div class="checkbox-group">
-            <input type="checkbox" id="remember_me" name="remember_me" value='1'[% ' checked' IF remember_me %]>
-            <label class="inline n" for="remember_me">[% loc('Keep me signed in on this computer') %]</label>
```

Comment:
It's tricky. It might mean they have to sign in less often. On a phone (assuming browser just left alone, never dying except on version upgrade), or on a desktop with modern 'sleep' rather than being shut down (or indeed, Chrome/Firefox "Continue Where I Left Off" restores session cookies, or at least certainly used to), a session cookie could easily end up lasting longer than a timed out cookie.

OTOH, 'keep me signed in' would set a timed cookie and the cookie expiry would move into the future as you used the site, so would be 4 weeks from last use. So perhaps it should always use a timed cookie rather than always use a session cookie; but either way we should pick one I think.

Reply:
Perhaps I've just become sensitised because I get irritated when I have to sign back in to Fantasy Bake Off *every week* (I mean really, set the cookie length to 8 days or something) or perhaps I'm just irritable 😃.

Do we have any clue how many people tick that box?

Reply:
In past four weeks, looks like we have 29k sessions in the db, of which 24.5k are session, 4.5k.
I think I'd be happy with a timed, extending, cookie. BBC doesn't mention anything on sign in, and says elsewhere in help their cookie keeps you signed in for two years (it is indeed a two year cookie).

Reply:
👍 to that idea.

Reply:
On checking, that is what this code is already doing, I think, so yay past me :)

Reply:
Oh, except if they come from a token, when it explicitly makes a session cookie in that case. Dunno if we should change that to match/be consistent.